sqlmodel/main.py

```diff
@@ -417,7 +417,7 @@ def get_column_from_field(field: ModelField) -> Column:
     nullable = not field.required
     index = getattr(field.field_info, "index", Undefined)
     if index is Undefined:
-        index = True
+        index = False
```

Comment:
This can be simplified:

```diff
- index = getattr(field.field_info, "index", Undefined)
-     if index is Undefined:
-         index = False
+ index = getattr(field.field_info, "index", False)
```

Reply:
That's not the same. The default for index in `field_info` is `Undefined`:
https://github.com/tiangolo/sqlmodel/blob/af03df88acda8c5fb3e0e755b0ab0517a2c8aabb/sqlmodel/main.py#L154

Reply:
Oh, my bad. I looked at it too narrowly.